sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -64,6 +85,53 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
         val tableIdent = extractTableIdent(nameParts)
         RefreshTable(tableIdent)
 
+      case Token("TOK_CREATEDATABASE", Token(databaseName, Nil) :: createDatabaseArgs) =>
+        val Seq(
+          allowExisting,
```

Comment:
When i see a parameter that is named `allowExists` I think the existence of the current table is allowed, and that as a consequence the current create table command is going to overwrite the existing table. Why not name this `ifNotExists`?